client/src/features/project/components/ProjectSettingsUtils.tsx

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright 2023 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import cx from "classnames";
+
+import { faExclamationTriangle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+interface SettingRequiresKgProps {
+  className?: string;
+}
+export function SettingRequiresKg({ className }: SettingRequiresKgProps) {
+  const localClass = className ? cx(className) : "d-block mt-2";
```

Comment:
No need for `cx` here:

```suggestion
  const localClass = className ?? "d-block mt-2";
```